pkg/controller/directvolumemigration/pvcs.go

```diff
@@ -114,7 +118,9 @@ func (t *Task) createDestinationPVCs() error {
 			"destPersistentVolumeClaim", path.Join(destNs, pvc.Name),
 			"pvcStorageClassName", destPVC.Spec.StorageClassName,
 			"pvcAccessModes", destPVC.Spec.AccessModes,
-			"pvcRequests", destPVC.Spec.Resources.Requests)
+			"pvcRequests", destPVC.Spec.Resources.Requests,
+			"pvcDataSource", destPVC.Spec.DataSource,
+			"pvcDataSourceRef", destPVC.Spec.DataSourceRef)
```

Comment:
nit: if it's always going to be nil, should we care logging it?

Reply:
Its more debug information, if it is NOT nil, there is a problem. The log should probably be debug instead of info.